src/org/labkey/test/pages/core/admin/ShowAdminPage.java

```diff
@@ -230,6 +237,7 @@ protected class ElementCache extends LabKeyPage.ElementCache
         protected WebElement systemMaintenanceLink = Locator.linkWithText("system maintenance").findWhenNeeded(this);
         protected WebElement systemPropertiesLink = Locator.linkContainingText("system properties").findWhenNeeded(this);
         protected WebElement viewsAndScriptingLink = Locator.linkWithText("views and scripting").findWhenNeeded(this);
+        protected WebElement creditsLink = Locator.linkWithText("credits").findElementOrNull(this);
```

Comment:
I'm guessing auto-complete picked the wrong method here.
```suggestion
        protected WebElement creditsLink = Locator.linkWithText("credits").findWhenNeeded(this);
```

Reply:
done